recipe/meta.yaml

```diff
@@ -50,10 +57,18 @@ outputs:
         - zstd {{ zstd }}
         - bzip2 {{ bzip2 }}
         - cli11
+        - simdjson >=3.3.0
+        - pkg-config
       run:
-        - libsolv >=0.7.24,<0.8.0a0
+        - libsolv >=0.7.30,<0.8.0a0
         - reproc-cpp
         - reproc
+        - fmt
+        - spdlog
+        - cpp-expected
+        - nlohmann_json
+        - simdjson
+        - yaml-cpp
     test:
       commands:
         - test -d ${PREFIX}/include/mamba                                   # [unix]
```

Comment:
Likewise, there are tests for their C++ module as well.
https://github.com/mamba-org/mamba/tree/main/libmamba/tests